cvat/apps/quality_control/quality_reports.py

```diff
@@ -2500,6 +2502,10 @@ def prepare_report_for_downloading(db_report: models.QualityReport, *, host: str
                     f"&serverID={ann_id['obj_id']}"
                 )
 
+    serialized_data["comparison_summary"]["frame_share_percent"] = (
+        serialized_data["comparison_summary"].pop("frame_share") * 100
+    )
```

Comment:
Why do this? I don't see why the report couldn't also contain the fraction.

Reply:
I talked to the annotators, they said they better like percents. Having both seems extraneous

Reply:
Added